components/events/fall-fest/UniversityDirectorySection.vue

```diff
@@ -38,111 +43,331 @@ import { GeneralLink } from '~/constants/appLinks'
+      }
+    },
+    {
+      title: 'Yale University',
+      image: '/images/events/fall-fest/connecticut.png',
+      startDate: 'October 3, 2022',
+      detail: 'In Person',
+      cta: {
+        label: 'Learn more here',
+        url: 'https://yaleqc.com/categories/events/',
+        segment: { cta: 'yale-university', location: 'fall-fest-page' }
+      }
+    },
+    {
+      title: 'QickStart : A Qiskit Powered Intro to Quantum Computing',
+      image: '/images/events/fall-fest/delhi.png',
+      startDate: 'October 3, 2022',
+      detail: 'In Person',
+      cta: {
+        label: 'Learn more here',
+        url: 'https://github.com/TheGupta2012/NSUT-Qiskit-Fall-Fest',
+        segment: { cta: 'qickstart-a-qiskit-powered-intro-to-quantum-computing', location: 'fall-fest-page' }
+      }
+    },
+    {
+      title: 'The University of Hong Kong',
+      image: '/images/events/fall-fest/hong_kong.png',
+      startDate: 'October 25, 2022',
+      detail: 'In Person',
+      cta: {
+        label: 'Learn more here',
+        url: '',
+        segment: { cta: 'the-university-of-hong-kong', location: 'fall-fest-page' }
+      }
+    },
+    {
+      title: 'Purdue University',
+      image: '/images/events/fall-fest/indiana.png',
+      startDate: 'October 3, 2022',
+      detail: 'In Person',
+      cta: {
+        label: 'Learn more here',
+        url: 'https://github.com/purduequaic/qiskit-fall-fest-22',
+        segment: { cta: 'purdue-university', location: 'fall-fest-page' }
+      }
+    },
+    {
+      title: 'Qiskit Fall Fest Kolkata Chapter',
+      image: '/images/events/fall-fest/kolkata.png',
+      startDate: 'October 17, 2022',
+      detail: 'Online',
+      cta: {
+        label: 'Learn more here',
+        url: 'https://ritajitmajumdar.github.io/',
+        segment: { cta: 'qiskit-fall-fest-kolkata-chapter', location: 'fall-fest-page' }
+      }
+    },
+    {
+      title: 'Qiskit Fall Fest Mexico-IPN',
+      image: '/images/events/fall-fest/mexico_city.png',
+      startDate: 'October 17, 2022',
+      detail: 'Online',
+      cta: {
+        label: 'Learn more here',
+        url: 'https://qisk.it/fallfest',
+        segment: { cta: 'qiskit-fall-fest-mexico-ipn', location: 'fall-fest-page' }
+      }
+    },
+    {
+      title: 'Case Western Reserve University',
+      image: '/images/events/fall-fest/ohio.png',
+      startDate: 'October 10, 2022',
+      detail: 'In Person',
+      cta: {
+        label: 'Learn more here',
+        url: 'https://github.com/Case-Quantum-Computing-Club/CQC-qiskit-fall-fest-22',
+        segment: { cta: 'qiskit-fall-fest-mexico-ipn', location: 'fall-fest-page' }
+      }
+    },
+    {
+      title: 'University of Chicago',
+      image: '/images/events/fall-fest/illinois.png',
+      startDate: 'October 10, 2022',
+      detail: 'In Person',
+      cta: {
+        label: 'Learn more here',
+        url: '',
```

Comment:
is there no URL for this one?

Reply:
@vabarbosa 

I can revert the package lock, good call out.

For empty `url` values, yes -- some schools are still in the process of getting the information to Brian. We suspect that going _live_ will be one way to give them more incentive get their info to him sooner.

🤞🏼

Reply:
cool. will approve